tests/integration_tests/run_constant_reprocessing_openmc/test.py

```diff
@@ -10,29 +10,34 @@
 import tables as tb
 import subprocess
 
+def _create_nuclide_map(node):
+    nuclides = list(map(bytes.decode, node.nuclide_map.col('nuclide')))
+    indices = node.nuclide_map.col('index')
+    return dict(zip(nuclides, indices))
+
 
 @pytest.fixture
 def setup(scope='module'):
     cwd = Path(__file__).parents[0].resolve()
     os.chdir(cwd)
     test_db = cwd / 'saltproc_runtime/saltproc_results.h5'
-    ref_db = cwd / 'msbr_reference_results.h5'
+    ref_db = cwd / 'pincell_reference_results.h5'
     atol = 3e-3
-    rtol = 4e-2
+    rtol = 5e-2
```

Comment:
Why are the tolerances set to these values?

Reply:
So, we are testing if things like fuel composition, keff, etc match. This test uses a very low particle count to ensure it runs in a reasonable amount of time, and the errors due to the low particle count and  be fairly large, so we need to set them sufficiently large so the test will pass.

Perhaps a better way of doing this test would be to define a reference error, and test that the test error is less than or equal to the reference error (this is what we do in the `run_*_no_reprocessing` integration tests).